docs/starters.yml

```diff
@@ -1777,3 +1777,20 @@
   features:
     - Generate nice documentation with Docz, in addition to generating your normal Gatsby site
     - Document your React components in .mdx files
+- url: https://gatsby-hello-friend.now.sh
+  repo: https://github.com/panr/gatsby-starter-hello-friend
+  description: A simple starter for Gatsby. That's it.
+  tags:
+    - Responsive
```

Comment:
```suggestion
```
Reponsive isn't really a tag we want to consider as all starters are responsive. This could live in features though.